Lua/Lua.sublime-syntax

```diff
@@ -692,9 +706,19 @@ contexts:
           scope: punctuation.section.group.end.lua
           pop: 1
         - include: reserved-word-expression-pop
+        - match: (?="(?:{{string_char_escape}}|[^\\"])*"\s*\)\s*:\s*format{{identifier_break}})
+          push: fmtstr-expression
+        - match: (?='(?:{{string_char_escape}}|[^\\'])*'\s*\)\s*:\s*format{{identifier_break}})
+          push: fmtstr-expression
+        - match: (?="(?:{{string_char_escape}}|[^\\"])*"\s*\)\s*:\s*(?:pack|unpack|packsize){{identifier_break}})
+          push: packstr-expression
+        - match: (?='(?:{{string_char_escape}}|[^\\'])*'\s*\)\s*:\s*(?:pack|unpack|packsize){{identifier_break}})
+          push: packstr-expression
```

Comment:
Those patterns expect pack/format strings to be one-liners.

But even expressions of concatted strings (e.g.: `("one %0s two" + "another %d"):format();`) are not scoped format string.

Other syntax definitions like python, don't restrict placeholder this way, as it is also valid to specify a format string in a separate variable.

```
fmt_str = "My %s format"
fmt_str:format("name")
```

I'd suggest to highlight placeholders everywhere or head for a more flexible approach using `branching` to scope all strings in a group followed by `:format()`, ... accordingly.